examples/kubernetes/README.md

```diff
@@ -0,0 +1,178 @@
+# optimum-habana-example-chart
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square)
+
+This folder contains a Dockerfile and [Helm chart](https://helm.sh) demonstrating how 🤗 Optimum Habana examples
+can be run using Intel® Gaudi® AI accelerator nodes from a vanilla Kubernetes cluster. The instructions below
+explain how to build the docker image and deploy the job to a Kubernetes cluster using Helm.
+
+## Requirements
+
+### Client System Requirements
+
+In order to build the docker images and deploy a job to Kubernetes you will need:
+
+* [Docker](https://docs.docker.com/engine/install/)
+* [Docker compose](https://docs.docker.com/compose/install/)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/) installed and configured to access a cluster
```

Comment:
```suggestion
* [kubectl](https://kubernetes.io/docs/tasks/tools/) installed and configured to access a cluster
* [Minikube](https://minikube.sigs.k8s.io/docs/start/) installed and configured to access a Kubernetes cluster on a single node
```

@dmsuehir would you provide a pointer for anyone using a single node and wants to test your PR to use Minikube.

```
If you are intending to run a Kubernetes cluster on a single node (locally), you will need a tool like [Minikube](https://minikube.sigs.k8s.io/docs/start/) to be installed first
```